nullaway/src/test/java/com/uber/nullaway/NullAwayAndroidTest.java

```diff
@@ -17,6 +17,7 @@
 
   private CompilationTestHelper compilationHelper;
 
+  @SuppressWarnings("CheckReturnValue")
```

Comment:
The documentation for "CheckReturnValue" is unusually unhelpful. But from what I understand, this is complaining that we are not "using" the result from `.newInstance(...)`, even though we are assigning it to a field, correct?

The weird thing is, this seems like perfectly idiomatic Error Prone test code, e.g.: https://github.com/google/error-prone/blob/master/test_helpers/src/test/java/com/google/errorprone/CompilationTestHelperTest.java#L45

Reply:
Yeah I don't fully understand why we need to suppress here either

Reply:
It's because `setArgs` is annoyingly marked as `@CheckReturnValue`, as they marked the entire package with that annotation